vermouth/data/quotes.txt

```diff
@@ -0,0 +1,32 @@
+A man must defend his home, his wife, his children, and his martini. -- Jackie Gleason
+
+I never go jogging, it makes me spill my martini. -- George Burns
+Shaken, not stirred. -- James Bond
```

Comment:
For consistency with the rest of the file, you could skip a line between these two quotes.